publicAPI/api_utils.py

```diff
@@ -33,26 +32,32 @@ def check_key(
         (bool) access allowed or not
 
     """
-    api_db = TinyDB(path.join(cache_path, api_file))
-
-    api_value = api_db.search(
-        Query().api_key == api_key
-    )
+    
+    #Connect to TinyMongoDB and use prosperAPI DB 
+    connection = TinyMongoClient(CACHE_PATH)
```

Comment:
((NOT FOR NOW -- design note))

Should instead change `def check_key()` to take connection handle rather than info?  Or perhaps add a `get_db()` and `close_db()` call to `config.py` so handles can be grabbed generically